scrapy/pipelines/media.py

```diff
@@ -16,6 +19,7 @@
 class MediaPipeline(object):
 
     LOG_FAILED_RESULTS = True
+    ALLOW_REDIRECTS = False
```

Comment:
I would prefer that no new class attribute is introduced.
Can this be implemented with a default `MEDIA_ALLOW_REDIRECTS` setting in `default_settings.py`?